test/tools/gen-operator-with-single-input.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+/* eslint guard-for-in: 0 */
+
+import path from 'path';
+import {softsign} from '../../src/softsign.js';
+import {gelu} from '../../src/gelu.js';
+import {Tensor} from '../../src/lib/tensor.js';
+import {utils} from './utils.js';
+
+(() => {
+  function computeBySingleInput(operatorName, input, options = {}) {
+    const operatorMappingDict = {
+      'gelu': gelu,
+      'softsign': softsign,
+    };
+    const inputTensor = new Tensor(input.shape, input.data);
+    const outputTensor = operatorMappingDict[operatorName](inputTensor, options);
+    return outputTensor.data;
+  }
+
+  const testDataFileName = path.basename(process.argv[2]);
+  const operatorString =
```

Comment:
Since you're already using Node's path library (isAbsolute below), would it be semantically clearer to use path.basename here? (https://nodejs.org/api/path.html#pathbasenamepath-suffix)